src/download.rs

```diff
@@ -195,6 +190,7 @@ impl SnapshotDownloader {
                 .await
                 .context(error::ListSnapshotBlocks { snapshot_id })?;
 
+
```

Comment:
Nit: extra unneeded line

Reply:
Thanks. I removed the line.

Reply:
Thanks Zac. I have squashed the commits.